src/main/java/com/turntabl/api/service/Data.java

```diff
@@ -0,0 +1,49 @@
+package com.turntabl.api.service;
+
+import com.turntabl.api.domain.Product;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class Data {
```

Comment:
This is an unusual naming, we should probably call this `productDemoData` to be clear